prow/cmd/deck/static/prow/prow.ts

```diff
@@ -739,15 +725,41 @@ function redraw(fz: FuzzySearch, pushState: boolean = true): void {
 function createRerunCell(modal: HTMLElement, rerunElement: HTMLElement, prowjob: string): HTMLTableDataCellElement {
     const c = document.createElement("td");
     c.appendChild(createRerunProwJobIcon(modal, rerunElement, prowjob, rerunCreatesJob, csrfToken));
-    c.classList.add("icon-cell");
     return c;
 }
 
+function createLogCell(build: ProwJob, buildUrl: string): HTMLTableDataCellElement {
+    const { agent, job, pod_spec } = build.spec
+    const { pod_name, build_id } = build.status
+
+    if ((agent === "kubernetes" && pod_name) || agent !== "kubernetes") {
+        const logIcon = icon.create("description", "Build log");
+        if (pod_spec == null || pod_spec.containers.length <= 1) {
+            logIcon.href = `log?job=${job}&id=${build_id}`;
+        } else {
+            // this logic exists for legacy jobs that are configured for gubernator compatibility
+            const buildIndex = buildUrl.indexOf('/build/');
+            if (buildIndex !== -1) {
+                const gcsUrl = `${window.location.origin}/view/gcs/${buildUrl.substring(buildIndex + '/build/'.length)}`;
+                logIcon.href = gcsUrl;
+            } else if (buildUrl.includes('/view/')) {
+                logIcon.href = buildUrl;
+            } else {
+                logIcon.href = `log?job=${job}&id=${build_id}`;
+            }
+        }
+        const c = document.createElement("td");
+        c.appendChild(logIcon);
+        return c;
+    }
+    
+    return cell.text("");
```

Comment:
`dashCell`, please. :)

Reply:
I opted leaving this one as an empty string, because replacing an icon with a dash doesn't seem as aesthetically pleasing to me.